PWGDQ/Core/MCSignalLibrary.h

```diff
@@ -159,6 +164,131 @@ MCSignal* o2::aod::dqmcsignals::GetMCSignal(const char* name)
     signal = new MCSignal(name, "dielectron from a photon conversion from a pi0", {prong, prong}, {1, 1});
     return signal;
   }
+
+  //LMEE single signals
+  if (!nameStr.compare("eFromPizero")) {
+    MCProng prong(2, {11, 111}, {true, true}, {false, false}, {0, 0}, {0, 0}, {false, false});
+    signal = new MCSignal(name, "Electrons from pi0 decays", {prong}, {-1});
+    return signal;
+  }
+  if (!nameStr.compare("eFromEta")) {
+    MCProng prong(2, {11, 221}, {true, true}, {false, false}, {0, 0}, {0, 0}, {false, false});
+    signal = new MCSignal(name, "Electrons from eta decays", {prong}, {-1});
+    return signal;
+  }
+  if (!nameStr.compare("eFromEtaPrime")) {
+    MCProng prong(2, {11, 331}, {true, true}, {false, false}, {0, 0}, {0, 0}, {false, false});
+    signal = new MCSignal(name, "Electrons from eta' decays", {prong}, {-1});
+    return signal;
+  }
+  if (!nameStr.compare("eFromRho")) {
+    MCProng prong(2, {11, 113}, {true, true}, {false, false}, {0, 0}, {0, 0}, {false, false});
+    signal = new MCSignal(name, "Electrons from rho decays", {prong}, {-1});
+    return signal;
+  }
+  if (!nameStr.compare("eFromOmega")) {
+    MCProng prong(2, {11, 223}, {true, true}, {false, false}, {0, 0}, {0, 0}, {false, false});
+    signal = new MCSignal(name, "Electrons from omega decays", {prong}, {-1});
+    return signal;
+  }
+  if (!nameStr.compare("eFromPhi")) {
+    MCProng prong(2, {11, 333}, {true, true}, {false, false}, {0, 0}, {0, 0}, {false, false});
+    signal = new MCSignal(name, "Electrons from phi decays", {prong}, {-1});
+    return signal;
+  }
+  if (!nameStr.compare("eFromJpsi")) {
+    MCProng prong(2, {11, 443}, {true, true}, {false, false}, {0, 0}, {0, 0}, {false, false});
+    signal = new MCSignal(name, "Electrons from jpsi decays", {prong}, {-1});
+    return signal;
+  }
+  if (!nameStr.compare("eFromPsi2S")) {
+    MCProng prong(2, {11, 100443}, {true, true}, {false, false}, {0, 0}, {0, 0}, {false, false});
+    signal = new MCSignal(name, "Electrons from psi2s decays", {prong}, {-1});
+    return signal;
+  }
+  if (!nameStr.compare("eFromSM")) {
```

Comment:
As for my comments above, please change "SM" to something else. Maybe "eFromLMeeCocktailParticles" or "eFromLMeeSpecialSources" ?